### DISCUSSION TITLE
Sync users table with other apps

### DISCUSSION THREAD

**Discussion by joeytzy** [Budibase/budibase#13987] [Help] [answered]
Hi, I have set up multiple app in budibase and linking it together through navigations and shared external databases. All except for the users table for each of the app. I am thinking of making a profile page where the user can update their information (first name, last name, etc...) but I have no idea how to sync the updated users table across all the other apps within the budibase. 

So I was wondering if there are any methods that allow me to sync or update all the users table when a row is updated in one app. Basically, if the user update their info in one app, other apps will see the changes as well

**Comment by ConorWebb96** [comment, marked as answer]
Hey @joeytzy,

The Budibase user's table isn't editable. Its a read only table that cannot be amended.

If you wish to add additional details create a new table called userDetails or something similar to that affect. You can then associate this additional table to your logged in users via the [single user](https://docs.budibase.com/docs/users-1) column. Then with a filter + the currentUser bindings you can ensure that only the logged in user can amend their details.

As for the first + last name and email seen in the users table, this can be edited by the user via logging into the Budibase portal and amending their details that way.

I hope the above makes sense.

**Reply by joeytzy** [reply]
so for the users table, I've managed to make an update form that allows me to update the data (first, last, email) in the users table. Not sure if its advisable though but Im still figuring out how to update the users table in another app maybe with budibase api

**Reply by ConorWebb96** [reply]
Hey @joeytzy,

You shouldn't be able to change the details within the users table. I've created an issue [here](https://github.com/Budibase/budibase/issues/13994) to prevent this in the future. The suggestion I gave you would be the best way of adding additional details for a user within your app.

I hope this makes sense.
